apps/standalone/src/app/components/AppLayout/AppToolbar.tsx

```diff
@@ -59,26 +64,33 @@ const UserDropdown: React.FC<UserDropdownProps> = ({ children, username = 'User'
 const AppToolbar = () => {
   const { t } = useTranslation();
   const [preferencesModalOpen, setPreferencesModalOpen] = React.useState(false);
-  const auth = useAuth();
+  const { username, enabled } = React.useContext(AuthContext);
```

Comment:
```suggestion
  const { username, authEnabled } = React.useContext(AuthContext);
```

for better readability